pallets/capacity/src/lib.rs

```diff
@@ -831,11 +830,16 @@ impl<T: Config> Pallet<T> {
 		staker: &T::AccountId,
 		proposed_amount: BalanceOf<T>,
 	) -> BalanceOf<T> {
+		let unlocks = UnstakeUnlocks::<T>::get(staker).unwrap_or_default();
+
+		let unthawed_unstaked_balance =
+			unthawed_unlock_chunks_total::<T>(&unlocks, CurrentEpoch::<T>::get());
 		let freezable_balance = T::Currency::balance_freezable(staker);
 		let current_staking_balance =
 			StakingAccountLedger::<T>::get(staker).unwrap_or_default().active;
 		let stakable_amount = freezable_balance
 			.saturating_sub(current_staking_balance)
+			.saturating_sub(unthawed_unstaked_balance)
```

Comment:
@shannonwells I updated this so that it now includes thawed unlock chunks in the stakeable balance. Also updated unit tests accordingly.

However, there is one side-effect of this that we may want to consider more thoroughly: this setup allows overlapping `CapacityFreeze` locks on the same tokens. I don't think it currently causes any weird effects, but it could be something we trip over in the future.